src/desktop/src/ui/components/input/Text.js

```diff
@@ -20,6 +20,8 @@ export default class Text extends React.PureComponent {
          * @param {string} value - Current input value
          */
         onChange: PropTypes.func.isRequired,
+        /** Limit max text length */
+        maxLength: PropTypes.integer,
```

Comment:
Whoops, this is actually called `number` 😬 

```suggestion
        maxLength: PropTypes.number,
```